project/AkkaDisciplinePlugin.scala

```diff
@@ -138,18 +145,16 @@ object AkkaDisciplinePlugin extends AutoPlugin {
    */
   val docs = Seq(
     scalacOptions ++= Seq(
-      // In docs, 'unused' variables can be useful for naming and showing the type
-      "-P:silencer:globalFilters=is never used",
-      // Import statements are often duplicated across multiple snippets in one file
-      "-P:silencer:globalFilters=Unused import",
-      // We keep documentation for this old API around for a while:
-      "-P:silencer:globalFilters=in object Dns is deprecated",
-      "-P:silencer:globalFilters=in class Dns is deprecated",
-      // Because we sometimes wrap things in a class:
-      "-P:silencer:globalFilters=The outer reference in this type test cannot be checked at run time",
-      // Because we show some things that are deprecated in
-      // 2.13 but don't have a replacement that was in 2.12:
-      "-P:silencer:globalFilters=deprecated \\(since 2.13.0\\)"
-    )
-  )
+        // In docs, 'unused' variables can be useful for naming and showing the type
+        "-P:silencer:globalFilters=is never used",
+        // Import statements are often duplicated across multiple snippets in one file
+        "-P:silencer:globalFilters=Unused import",
+        // We keep documentation for this old API around for a while:
+        "-P:silencer:globalFilters=in object Dns is deprecated",
+        "-P:silencer:globalFilters=in class Dns is deprecated",
+        // Because we sometimes wrap things in a class:
+        "-P:silencer:globalFilters=The outer reference in this type test cannot be checked at run time",
+        // Because we show some things that are deprecated in
+        // 2.13 but don't have a replacement that was in 2.12:
+        "-P:silencer:globalFilters=deprecated \\(since 2.13.0\\)"))
```

Comment:
It seems Scala 2.13 doesn't like these settings - are there equivalent settings to configure 'nowarn'?

Reply:
I will fix it